api/src/test/java/net/kyori/adventure/text/BlockNBTComponentTest.java

```diff
@@ -95,16 +95,16 @@ void testLocalPosParsing() {
   @Test
   void testAbsoluteWorldPosParsing() {
     assertEquals(
-      BlockNBTComponent.WorldPos.worldPos(BlockNBTComponent.WorldPos.Coordinate.absolute(4), BlockNBTComponent.WorldPos.Coordinate.absolute(5), BlockNBTComponent.WorldPos.Coordinate.absolute(6)),
-      BlockNBTComponent.Pos.fromString("4 5 6")
+      BlockNBTComponent.WorldPos.worldPos(BlockNBTComponent.WorldPos.Coordinate.absolute(4), BlockNBTComponent.WorldPos.Coordinate.absolute(-5), BlockNBTComponent.WorldPos.Coordinate.absolute(6)),
+      BlockNBTComponent.Pos.fromString("4 -5 6")
     );
   }
 
   @Test
   void testRelativeWorldPosParsing() {
     assertEquals(
-      BlockNBTComponent.WorldPos.worldPos(BlockNBTComponent.WorldPos.Coordinate.relative(7), BlockNBTComponent.WorldPos.Coordinate.relative(83), BlockNBTComponent.WorldPos.Coordinate.relative(900)),
-      BlockNBTComponent.Pos.fromString("~7 ~83 ~900")
+      BlockNBTComponent.WorldPos.worldPos(BlockNBTComponent.WorldPos.Coordinate.relative(-7), BlockNBTComponent.WorldPos.Coordinate.relative(83), BlockNBTComponent.WorldPos.Coordinate.relative(900)),
+      BlockNBTComponent.Pos.fromString("~-7 ~83 ~900")
     );
```

Comment:
Please don't modify the tests to change the numbers - add new tests for negative numbers.